### DISCUSSION TITLE
Test failing with error message: not found within 10000 ms

### DISCUSSION THREAD

**Discussion by Ananda0488** [getgauge/taiko#2247] [General]
My test is failing in the following line:
await waitFor($(ServiceName), {maxTime: 180000})

Error: Waiting Failed: Element '[object Object]' not found within 10000 ms

Can any one please tell me, from where is it getting 10000 ms?

**Comment by NivedhaSenthil** [comment]
The default timeout of wait is 10000 ms, there is no option like maxTime and hence that would have not been honored. For giving custom time the api can be simply called as  `await waitFor($(ServiceName), 180000)`

**Reply by Ananda0488** [reply]
ok... thank you... :)

**Comment by Pekshika** [comment]
when content within iframe is not loaded getting WARNING: Taiko cannot highlight hidden elements. and above error, how to solve this?
